### DISCUSSION TITLE
Request Approved and received email

### DISCUSSION THREAD

**Discussion by AmirAbbas-Mashayekhi** [community/community#147775] [GitHub Education]
Hi, I did everything necessary and my request was approved 3 days ago. Today I received an email stating that my information was reviewed and accepted. But my student pack benefits are not yet activated.

**Comment by davevad93** [comment]
Hi @AmirAbbas-Mashayekhi ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @AmirAbbas-Mashayekhi was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640, #145995

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by AmirAbbas-Mashayekhi** [reply]
Thank you very much🫂

**Reply by davevad93** [reply]
You're welcome @AmirAbbas-Mashayekhi 👍
